Dockerfile-test

```diff
@@ -1,2 +1,5 @@
 FROM fxa-profile-server:build
+USER root
+RUN apk add --repository http://dl-cdn.alpinelinux.org/alpine/v3.5/community/ --no-cache --virtual .build-deps git python make g++
```

Comment:
It isn't absolutely necessary to add the binary deps here, but if test dependencies have binary node modules, this would allow them to build.

Reply:
yeap 👍